java/src/test/scala/com/example/PoC.scala

```diff
@@ -45,7 +56,6 @@ object Poc extends IOApp.Simple {
       Attribute(fish, List("one", "two", "red", "blue")),
       Attribute(numbers, List(1L, 2L, 3L, 4L))
     )
-    provider = otel4j.getSdkMeterProvider
-    _ <- IO.println(provider.forceFlush())
+    _ <- jMetricReader.collectAllMetrics().asScala.toList.traverse(IO.println)
```

Comment:
Not that it matters here, but I think this should be `traverse_`.  I like advertising its existence so people know it's there when it matters. :smile:
```suggestion
    _ <- jMetricReader.collectAllMetrics().asScala.toList.traverse_(IO.println)
```